src/runtime.js

```diff
@@ -429,11 +437,11 @@ var Runtime = {
 #endif
 };
 
-Runtime.stackAlloc = unInline('stackAlloc', ['size']);
-Runtime.staticAlloc = unInline('staticAlloc', ['size']);
-Runtime.dynamicAlloc = unInline('dynamicAlloc', ['size']);
-Runtime.alignMemory = unInline('alignMemory', ['size', 'quantum']);
-Runtime.makeBigInt = unInline('makeBigInt', ['low', 'high', 'unsigned']);
+Runtime.stackAlloc = unInline('stackAlloc', ['size'], true);
```

Comment:
`unInline` was written to always craft the function in the form of `'var ret = ' + string + '; return ret;'`, but the modified `dynamicAlloc` function is not an assignable expression that would fit there, but is  a whole function of its own.